packages/office-ui-fabric-react/src/components/ProgressIndicator/ProgressIndicator.base.tsx

```diff
@@ -93,23 +107,42 @@ export class ProgressIndicatorBase extends BaseComponent<IProgressIndicatorProps
       transition: percentComplete !== undefined && percentComplete < ZERO_THRESHOLD ? 'none' : undefined
     };
 
-    const ariaValueMin = percentComplete !== undefined ? 0 : undefined;
-    const ariaValueMax = percentComplete !== undefined ? 100 : undefined;
-    const ariaValueNow = percentComplete !== undefined ? Math.floor(percentComplete!) : undefined;
+    const percentCompleteAriaText = ariaValueText ? ariaValueText : this.state.percentComplete + '%';
+    const indeterminateAriaText = ariaValueText ? ariaValueText : 'Working on it';
```

Comment:
hmm. Hardcoded english string. We try to avoid these and ask them as prop input. I think we should be at least starting to include localized strings in resjson files.

Reply:
ariaValueText is the prop. Would you recommend having different props for percentCompleteAriaText and indeterminateAriaText  to handle this? The goal was to have a default aria-text. Let me know how you want me to proceed.

Reply:
@dzearing Let me know how you want me to handle this string situation. Seems like either:
1. 2 different props for indeterminate and determinate `ariaValueText`, the indeterminate one defaults to what I have up there
2. Just leave the one prop, no defaults (would have to confirm with @betrue-final-final on this one)
3. Leave as is and deal with localized strings in a separate effort